kinetic/distribution.yaml

```diff
@@ -553,6 +553,24 @@ repositories:
       url: https://github.com/ros-gbp/bfl-release.git
       version: upstream
     status: maintained
+  binpicking_utils:
+    doc:
+      type: git
+      url: https://github.com/durovsky/binpicking_utils.git
+      version: git
```

Comment:
here you need to give a branch name or a version number (same in the source section). see below for how other packages populate that field